Tools/jit/_stencils.py

```diff
@@ -181,6 +182,9 @@ class Stencil:
     body: bytearray = dataclasses.field(default_factory=bytearray, init=False)
     holes: list[Hole] = dataclasses.field(default_factory=list, init=False)
     disassembly: list[str] = dataclasses.field(default_factory=list, init=False)
+    trampolines: dict[str | None, int] = dataclasses.field(
```

Comment:
I don't think this can be `None`:

```suggestion
    trampolines: dict[str, int] = dataclasses.field(
```

(You may need to `assert hole.symbol is not None` at the top of `emit_aarch64_trampoline` to make `mypy` happy... not sure.)